src/pages/billing/CreateGCPBillingProject.js

```diff
@@ -29,6 +32,9 @@ const CreateGCPBillingProject = ({
           onChange: v => {
             setBillingProjectName(v)
             setBillingProjectNameTouched(true)
+            if (billingProjectWizard) {
```

Comment:
Instead of passing `billingProjectWizard`, can you pass in (optional) event names for `billingProjectNameEntered` and `billingAccountChosen` and then emit the events if those event names are specified?

Although perhaps we should just always admit the events regardless of whether it is the wizard vs. the regular modal, but change their event strings to be more generic.

Reply:
Also, calling this "buttonName" is confusing because there is no button.

Reply:
Talking to @admull, we would rather the even names be descriptive than "buttonName" be passed. So for instance, this event name could be something like

`billing:creation:gcpProjectNameEntered`, and the one below could be `billing:creation:gcpBillingAccountSelected`.